common/src/main/java/io/apicurio/registry/rest/v2/beans/EditableMetaData.java

```diff
@@ -100,13 +100,4 @@ public void setProperties(Map<String, String> properties) {
         this.properties = properties;
     }
 
-    @Override
-    public String toString() {
```

Comment:
I think all this toString methods were there for the audit logs, maybe it's time to add the lombok toString annotation we discussed about once

Reply:
Yes, they were added to be used by audit logging.

Reply:
Yes I did realize that.  I will be adding the lombok toString annotation.